plugins/psv-extractor/src/psv.extractor.ts

```diff
@@ -8,45 +8,72 @@ import Papa, { ParseResult } from 'papaparse'
 
 export class PsvExtractor extends AbstractExtractor {
   private readonly _options: {
-    //add if needed
+    delimiter?: ',' | '\t' | '|' | ';'
```

Comment:
⛳ Personally I think using names here is more readable than symbols. E.g. `delimiter?: "comma" | "tab" | "pipe" | "semicolon"`